Testing/functional/e2e/cypress/integration/report/report.js

```diff
@@ -42,7 +83,7 @@ describe('Reports', () => {
             .should('be.visible');
 
         cy.get('[data-testid=genericMessageText]')
-            .should('have.text', ' The file that you are downloading contains personal information. If you are on a public computer, please ensure that the file is deleted before you log off. ');
+            .should('have.text', sensitiveDocText);
```

Comment:
This works but may be a little brittle... We could probably assert that the element exists rather than what the text says.